crates/fuel-core-nats/src/main.rs

```diff
@@ -12,7 +12,7 @@ pub struct Cli {
     )]
     nats_url: String,
     /// The NKEY seed. It is usually prefixed with an 'S'
-    #[arg(long, value_name = "NKEY", env = "NATS_NKEY")]
+    #[arg(long, value_name = "NKEY_SEED", env = "NATS_NKEY_SEED")]
     nats_nkey: Option<String>,
```

Comment:
We can delegate to `clap` for brevity

```suggestion
    #[arg(long, value_name = "NKEY_SEED", env = "NATS_NKEY_SEED")]
    nats_nkey: String,
```